src/middlewares/auth.js

```diff
@@ -0,0 +1,15 @@
+const jwt = require('jsonwebtoken');
+
+const isAuth = (req, res, next) => {
+  if (req.cookies) {
+    const { token } = req.cookies;
+    if (!token) {
+      res.sendStatus(401);
+    } else {
+      req.user = jwt.verify(token, process.env.JWT_SECRET);
```

Comment:
surround the `jwt.verify` with a `try/catch`, and in `catch` block, send a message to the user, about an invalid/expired token.
something like this: 

```javascript
       try {
        req.user = jwt.verify(token, process.env.JWT_SECRET);
      } catch (err) {
        return next(new ErrorResponse('Invalid/expired token', 401));
      }
```